src/realm/sync/noinst/client_impl_base.hpp

```diff
@@ -900,18 +892,11 @@ class ClientImpl::Session {
     const std::string& get_virt_path() const noexcept;
 
     const std::string& get_realm_path() const noexcept;
-    DBRef get_db() const noexcept;
 
-    /// The implementation need only ensure that the returned reference stays valid
-    /// until the next invocation of access_realm() on one of the session
-    /// objects associated with the same client object.
-    ///
-    /// This function is always called by the event loop thread of the
-    /// associated client object.
-    ///
-    /// This function is guaranteed to not be called before activation, and also
-    /// not after initiation of deactivation.
-    ClientReplication& access_realm();
```

Comment:
`access_realm()` should have gone away in #4839, as it was part of the MRU cache that the sync client used to have. These days it was just confusing and weird (and claimed to do something quite different from what it did).